components/AvalancheForecastZonePolygon.tsx

```diff
@@ -63,14 +63,17 @@ export const AvalancheForecastZonePolygon: React.FunctionComponent<AvalancheFore
     coordinates: toLatLngList(zone.geometry),
     strokeColor: selected ? highlight.toString() : outline.toString(),
     strokeWidth: selected ? 4 : 2,
-    tappable: true,
+    tappable: onPress !== undefined,
     zIndex: selected ? 1 : 0,
     onPress: (event: PolygonPressEvent) => {
-      onPress(zone);
-      // By calling stopPropagation, we prevent this event from getting passed to the MapView's onPress handler,
-      // which would then clear the selection
-      // https://github.com/react-native-maps/react-native-maps/issues/1132
-      event.stopPropagation();
+      if (onPress) {
+        onPress(zone);
+
+        // By calling stopPropagation, we prevent this event from getting passed to the MapView's onPress handler,
```

Comment:
Interestingly the behavior I noted was that if `tappable` was set, the map *would* get the event but the coordinates were `undefined`. Stopping propagation is one way to get rid of that, another would be to check that the location of the tap was not in a zone, in the map handler.

Reply:
It seems like the comment in the code describes what's going on correctly, are you saying something should change?

Reply:
Not quite - in my call for the polygon, I don't set an `onPress` and this code does not fire, but the polygon somehow still eats the substantive part of the event.

Reply:
Ah ok. I still think it's semi-surprising that the library propagates the events - it definitely doesn't seem like a sane default for them.